src/resources/postcss/tickets-commerce/admin/orders/table.pcss

```diff
@@ -104,35 +105,48 @@ body.wp-admin.post-type-tec_tc_order {
 	}
 }
 
-.tribe-external-link {
+.tribe-dashicons {
 	&--code {
 		font-family: monospace;
 	}
 
-	&:after {
-		content: url("../../../../images/tribe-external-link.svg");
```

Comment:
Aren't we using this for the Gateway link - or are we using the dashicon now?
<img width="260" alt="Screenshot 2024-08-13 at 10 18 24 AM" src="https://github.com/user-attachments/assets/2e9666b1-9bab-4191-9013-87f4d224f272">

Reply:
We the design is using the dashicon so i updated it here as well.